src/pages/lib/analytics/getting-started/q/platform/[platform].mdx

```diff
@@ -162,3 +162,21 @@ Congratulations! Now that you have Analytics' backend provisioned and Analytics
 - [Record Events](/lib/analytics/record)
 - [Track Sessions](/lib/analytics/autotrack)
 - [Identify User](/lib/analytics/identifyuser)
+
+<InlineFilter filters={['react-native']}>
+
+## Known Issues
+
+When importing alternative service providers listed below, instead of the default Pinpoint provider:
+* Kinesis (`aws-amplify/analytics/kinesis`)
+* Kinesis Data Firehose (`aws-amplify/analytics/kinesis-firehose`)
+* Personalize Event (`aws-amplify/analytics/personalize`)
```

Comment:
Should this just be Personalize?

```suggestion
* Personalize (`aws-amplify/analytics/personalize`)
```

Reply:
The AWS service(data plane) it uses is called Personalize Event: https://docs.aws.amazon.com/personalize/latest/dg/API_UBS_PutEvents.html